README.md

```diff
@@ -1,24 +1,154 @@
+**[Budget App]** A budget app is a tool designed to help individuals or households manage their finances by tracking income and expenses. With features like expense categorization, and expenses tracking, budget apps can be a useful way to improve financial management and achieve financial goals.
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+Built with Ruby on Rails 
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Add your Categories for expenses
+- Add new transactions for each category
+- Get the total per category for each category
+- Can have many users to use the app
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo Link](https://rails-budget-app-s2ql.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: ``` Ruby ```, ``` Rails ``` and a Database such as PostgreSQL
+
+### Setup
+
+Clone this repository to your desired folder.
+
+### Install
+
+Install this project with:
+
+```
+  bundle install
+  rails db:migrate
+```
+run the following commands, to be able to use Bootstap and JavsScript
+  rails javascript:install:esbuild
+  rails css:install:bootstrap
+### Usage
+
+To run the project, execute the following command:
+
+```
+  pg_ctl start
+  rails server
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+👤 **Shaker AbuDrais**
+
+- GitHub: [@githubhandle](https://github.com/shakerAbuDrais)
+- LinkedIn: [LinkedIn]( https://www.linkedin.com/in/shaker-abady/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add the rest of the necessary features for a functional blog.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a ⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thank you to microverse for setting us on this journey.
+
```

Comment:
## [OPTIONAL]

- [ ] I think it would be nice to follow the template [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including: `Colors` `Typography: font face, size, and weight.` `Layout: composition and space between elements

- [ ] Kindly make sure you give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) here in the Acknowledgments section, as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).